internal/handlers/user.go

```diff
@@ -443,7 +443,7 @@ func mapUserInfoWoRole(user *ent.User) *models.UserInfo {
 		ID:                      &userID,
 		IsReadonly:              &user.IsReadonly,
 		Login:                   &user.Login,
-		Name:                    &user.Name,
+		Name:                    user.Name,
```

Comment:
Is it satisfy the struct field type? (pointer vs value)

Reply:
The Name in the UserInfo struct was already a pointer, so yes